docker-compose.yml

```diff
@@ -238,9 +239,11 @@ services:
     labels:
       - "logging=true"
   case-documents-sync:
-    image: kanselarij/case-documents-sync-service:1.4.0
+    image: kanselarij/case-documents-sync-service:feature-max-body-size
     logging: *default-logging
     restart: always
+    environment:
+      MAX_BODY_SIZE: "500kb"
     labels:
```

Comment:
MAX_BODY_SIZE is also declared in the PR `feature-max-body-size`
Do we need both?